src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  let currentTab = tabs.find(tab => tab.id === selectedTabId);
```

Comment:
```suggestion
  const currentTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```
You can do something like this to avoid such a strange redeclaration